rpc/sip.go

```diff
@@ -54,6 +54,11 @@ func NewCreateSIPParticipantRequest(
 		features = append(features, livekit.SIPFeature_KRISP_ENABLED)
 	}
 
+	headers := req.Headers
+	if headers == nil {
+		headers = trunk.Headers
```

Comment:
You'll have to iterate over key-values and add them to merge maps properly. Here you're discarding one of the maps.

Reply:
I see. My assumption was that user would be having headers in either `CreateSIPParticipantRequest` or `SIPOutboundTrunkInfo`, and if they had in both it would be the same- hence just overriding in case one is nil. Will make the requested change